src/content/en/updates/2017/06/budget-api.md

```diff
@@ -11,14 +11,14 @@ description: The Budget API allows developers to perform background actions with
 
 The Push Messaging API enables us to send notifications to a user even when the
 browser is closed. Many developers want to be able to use this messaging to
-update and synchornise content without the browser being open, but the API has
+update and synchronize content without the browser being open, but the API has
```

Comment:
Wow - thanks for catching, I totally missed it, sorry about that. Agreed, it should be consistent in the doc.